### ISSUE TITLE
Invalid path assumptions on Linux

### ISSUE THREAD

**Issue by jvilk** [litixsoft/karma-detect-browsers#7]
Thanks for making this. I noticed that you assume that Linux browsers are in `/usr/bin`, but in many environments (such as Travis-CI), they are actually in `/usr/local/bin`.

Perhaps a better solution would be to use `which` to determine the path of an executable in the environment? The [`which`](https://github.com/npm/node-which) package might be useful for this.

**Comment by jvilk** [comment]
Here's the fix, as I imagine it.

**Current code:**

```{js}
// iterate over all browser paths
for (y = 0; y < paths; y++) {
    if (fs.existsSync(browserPaths[y]) || process.env[browser.ENV_CMD] || fs.existsSync(path.join('/', 'usr', 'bin', browserPaths[y]))) {
        // add browser when found in file system or when env variable is set
        result.push(browser.name);

        // set env variable on win32 when it does not exist yet
        if (process.platform === 'win32' && !process.env[browser.ENV_CMD]) {
            process.env[browser.ENV_CMD] = browserPaths[y];
        }

        break;
    }
}
```

**Updated code:**

```{js}
// iterate over all browser paths
for (y = 0; y < paths; y++) {
    try {
        if (fs.existsSync(browserPaths[y]) || process.env[browser.ENV_CMD] || which.sync(browserPaths[y])) {
            // add browser when found in file system or when env variable is set
            result.push(browser.name);
    
            // set env variable on win32 when it does not exist yet
            if (process.platform === 'win32' && !process.env[browser.ENV_CMD]) {
                process.env[browser.ENV_CMD] = browserPaths[y];
            }
    
            break;
        }
    } catch (e) {
        // which() failed to find the browser.
    }
}
```

I'd be happy to open a PR if this sounds interesting.

**Comment by 4kochi** [comment]
Looks good. Feel free to open a pull request.

**Comment by jvilk** [comment]
Done. See https://github.com/litixsoft/karma-detect-browsers/pull/9 . Check the Travis-CI tests; your module now detects additional browsers on Travis-CI! :smile:

**Comment by 4kochi** [comment]
Thx for the pull request. Released as version 2.0.2
